### PR TITLE
Update 🥒 tests to use `dockerfile_inline`

### PR DESCRIPTION
**Pull request by laurazard**
**What I did**

Update Cucumber tests to use `dockerfile_inline` when appropriate, for clarity (and coverage).

**(not mandatory) A picture of a cute animal, if possible in relation to what you did**

image

### CODE REVIEW

**Comment by ndeloof** [comment]
LGTM. Maybe we could have both ?

**Review by ndeloof — APPROVED** [review]

**Review by glours — APPROVED** [review]

**Comment by laurazard** [comment]
Yup! I changed both since the previous usecase is already thoroughly covered in the other tests, but we can add another cucumber scenario to make sure both are covered here. I'll TAL at it later.

**Review by milas — APPROVED** [review]

File: e2e/cucumber-features/ps.feature [hunk 1]
```diff
@@ -8,14 +8,12 @@ Background:
             image: test:latest
             build:
                 context: ./
+                dockerfile_inline |
+                    FROM golang:1.19-alpine
```
Comment (milas) [inline e2e/cucumber-features/ps.feature:12]:
```suggestion
                dockerfile_inline: |
                    FROM golang:1.19-alpine
```

**Review by milas — CHANGES_REQUESTED** [review]
Oops I take-backsies my approval, there's a typo in the YAML so the CI is failing

File: e2e/cucumber-features/ps.feature (hunk 1 above)
Comment (laurazard) [reply e2e/cucumber-features/ps.feature]: thanks! :)

**Review by milas — APPROVED** [review]

**Pull request merged**